Platform.Win32/Tests.Win32/Tests.Win32.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
```

Comment:
>Our assembly is AnyCPU, though.

Then that is a separate bug.

If you have an assembly with a dependency on native that **requires** it to be built as x86 or x64 it should never be built and shipped as _AnyCPU_